docs/tls/KUBERNETES-SECRETS.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes Secrets
+
+## Overview
+
+Kubernetes Secrets are used to provide the required certificates to NLK. There are two ways to create the Secrets:
+- Using `kubectl`
+- Using yaml files
+
+The filenames for the certificates created previously are required for both methods. The examples below assume the certificates were generated in `/tmp` 
+and follow the naming conventions in the documentation. 
+
+## Using `kubectl`
+
+The easiest way to create the secret is by using `kubectl`:
+
+```bash
+kubectl create secret tls -n nlk nlk-tls-{ca,server,client}-secret --cert=/tmp/{ca,server,client}.crt --key=/tmp/{ca,server,client}.key
```

Comment:
My ignorance on `kubectl` but does `{ca, server, client}` actually use all those names to create seperate secrets, or are you trying to denote that you'll need to run the command once for each file?  If so, although it's annoying I'd suggest providing all the discrete commands.  If not - please ignore me.